src/formatting.rs

```diff
@@ -74,6 +76,30 @@
 //! `format` or `f`        | [chrono docs](https://docs.rs/chrono/0.3.0/chrono/format/strftime/index.html#specifiers) for all options. | `'%a %d/%m %R'`
 //! `locale` or `l`        | Locale to apply when formatting the time                                                                  | System locale
 //!
+//!
+//! ## `duration`/`dur` - Format durations
+//!
+//! Argument                     | Description                                                                                      |Default value
+//! -----------------------------|--------------------------------------------------------------------------------------------------|------------------------------------------------------
+//! `hms`                        | Should the format be hours:minutes:seconds.milliseconds                                          | `false`
+//! `max_unit`                   | The largest unit to display the duration with (see below for the list of all possible units)     | hms ? `h` : `y`
+//! `min_unit`                   | The smallest unit to display the duration with (see below for the list of all possible units)    | `s`
+//! `units`                      | The number of units to display                                                                   | min(# of units between `max_unit` and `min_unit``, 2)
+//! `round_up`                   | Round up to the nearest minimum displayed unit                                                   | `true`
+//! `unit_space`                 | Should there be a space between the value and unit symbol (not allowed when `hms:true`)                | `false`
+//! `pad_with`                   | The character that is used to pad the numbers                                                    | hms ? `0` : ` ` (a space)
+//! `show_leading_units_if_zero` | If fewer than `units` are non-zero should leading numbers that have a value of zero be shown     | `true`
```

Comment:
I think something shorter would be nice :) Maybe `leading_zeroes`?

Reply:
I think you forgot to update the docs.

Reply:
Ah, because I forgot to hit save before commiting